activities/Stopwatch.activity/lib/tutorial.js

```diff
@@ -0,0 +1,127 @@
+define(["webL10n"], function (l10n) {
+	var tutorial = {};
+
+	tutorial.start = function() {
+		var steps = [
+			{
+				element: "",
+				orphan: true,
+				placement: "bottom",
+				title: l10n.get("TutoExplainTitle"),
+				content: l10n.get("TutoExplainContent")
+			},
+			{
+				element: "#fullscreen-button",
+				placement: "bottom",
+				title: l10n.get("TutoFullScreenButtonTitle"),
+				content: l10n.get("TutoFullScreenButtonContent")
+			},
+            {
```

Comment:
Avoid mixing indentation style in the same file.

Reply:
Indention Issue Fixed.